hydromt_wflow/wflow.py

```diff
@@ -876,176 +1015,187 @@ def setup_gauges(
 
         Adds model layers:
 
-        * **wflow_gauges** map: gauge IDs map from catchment outlets [-]
         * **wflow_gauges_source** map: gauge IDs map from source [-] (if gauges_fn)
         * **wflow_subcatch_source** map: subcatchment based on gauge locations [-] (if derive_subcatch)
-        * **gauges** geom: polygon of catchment outlets
         * **gauges_source** geom: polygon of gauges from source
         * **subcatch_source** geom: polygon of subcatchment based on gauge locations [-] (if derive_subcatch)
 
         Parameters
         ----------
-        gauges_fn : str, {"grdc"}, optional
-            Known source name or path to gauges file geometry file, by default "grdc".
-        source_gdf : geopandas.GeoDataFame, optional
-            Direct gauges file geometry, by default None.
+        gauges_fn : str, Path, gpd.GeoDataFrame, optional
+            Known source name or path to gauges file geometry file, by default None.
+
+            * Required variables if snap_uparea is True: ["uparea"]
```

Comment:
Good one! I put it in the docstrings description, we don't really have a good format yet on where to put units. "uparea" is chosen because this is the HydroMT name / unit convention so like for MERIT Hydro or DEM data, the user should use the data catalog to rename or convert units

Reply:
indeed fine to handle this renaming through the data catalog!